src/member/models.py

```diff
@@ -151,6 +151,9 @@ def add_contact_information(self, type, value, force_update=False):
         updated. Otherwise, it should create a new one.
         """
         created = False
+        if value is None:
+            value = ''
+
         if force_update or value is not '':
             contact, created = Contact.objects.update_or_create(
```

Comment:
Hmm it is to fix test who broke with error message indicate we try to insert NULL value for the field 'value' inside the database.
But sometimes we can have force_value = true with value=None, who raise the exception. So i consider None as an empty string to not raise the error.